scormxblock/static/sass/scormxblock.scss

```diff
@@ -0,0 +1,193 @@
+/* CSS for ScormXBlock */
+.close-navigation {
+    margin-bottom: 10px;
+
+    @media (max-width: 600px) {
+        margin: 0 0 30px 15px;
+    }
+
+    &:focus {
+        box-shadow: none;
+    }
+}
+
+.navigation-links {
+    text-align: center;
+}
+
+.container-fluid {
+    padding: 0 !important;
+}
+
+.full-screen-on {
+    display: block;
+    float: right;
+
+    @media (max-width: 600px) {
+        margin-right: 15px;
+    }
+}
+
+.scormxblock_block {
+    display: flex;
+    margin-bottom: 10px;
+
+    @media (max-width: 600px) {
+        flex-direction: column;
+    }
+
+    .js-scorm-block {
+        width: 100%;
+
+        @media (max-width: 600px) {
+            margin-bottom: 30px;
+        }
+    }
+
+    .scorm_object {
+        border: none;
+        margin-left: 15px;
+
+        @media (max-width: 600px) {
+            margin-left: 0;
+        }
+    }
+
+    span.setting-input-file {
+        padding-left: 25%;
+        margin-left: 14px;
+        padding-top: 5px;
+
+        span {
+            font-weight: bold;
+        }
+    }
+
+    .full-screen-scorm {
+        position: fixed;
+        border: none;
+        overflow: hidden;
+        top: 0;
+        right: 0;
+        bottom: 0;
+        left: 0;
+        width: 100%;
+        height: 100%;
+        z-index: 9999999999;
+        background-color: #fff;
+
+        .full-screen-off {
+            display: block;
+            margin: 10px;
+            position: absolute;
+            right: 20px;
+        }
+
+        .full-screen-on {
+            display: none;
+        }
+
+        .scorm_object {
+            position: fixed;
+            border: none;
+            overflow: hidden;
+            top: 55px;
+            right: 0;
+            bottom: 0;
+            left: 0;
+            width: 100%;
+            height: calc(100% - 65px);
+            z-index: 9999999;
+        }
+    }
+
+    .full-screen-off {
+        display: none;
+    }
+
+    .scorm-structure {
+        width: 300px;
+        margin-right: 0;
+
+        @media (max-width: 600px) {
+            padding: 0 25px 15px 15px;
+            width: 90%;
+        }
+    }
+
+    .scorm-structure-navigation {
+        max-width: 200px;
+        height: 450px;
+        overflow: auto;
+        margin: 0;
+        padding-left: 0 !important;
+
+        @media (max-width: 600px) {
+            height: auto;
+            max-width: 100%;
+        }
+
+        &-head {
+            color: #333;
+            font: 600 14px/20px Montserrat,"Helvetica Neue",Helvetica,Arial,sans-serif;
```

Comment:
I'm not sure that we need to override font-family because it became not adaptive to the general platform font.

Reply:
Done